torch/nn/attention/__init__.py

```diff
@@ -69,10 +69,13 @@ def sdpa_kernel(backends: List[SDPBackend]):
     This context manager can be used to select which backend to use for scaled dot product attention.
     Upon exiting the context manager, the previous state of the flags will be restored, enabling all backends.
     """
-    assert backends is None or isinstance(
-        backends, list
+    assert isinstance(
+        backends, (list, SDPBackend)
     ), "Backend must be an instance of SDPBackend or a list of SDPBackend instances"
 
+    if isinstance(backends, SDPBackend):
+        backends = [backends]
+
```

Comment:
tiny nit: I think this part is actually not necessary since on 79 we do `set(backends)` anyway :)

Reply:
The problem I had when I tried without that section is that set() expects an iterable as an argument.

Reply:
oh oof my bad, sorry!